core/src/test/java/org/axonframework/queryhandling/SubscriptionQueryTest.java

```diff
@@ -46,8 +47,11 @@
  */
 public class SubscriptionQueryTest {
 
-    private final SimpleQueryBus queryBus = new SimpleQueryBus();
-    private final ChatQueryHandler chatQueryHandler = new ChatQueryHandler(queryBus);
+    private final SimpleQueryUpdateEmitter queryUpdateEmitter = new SimpleQueryUpdateEmitter();
```

Comment:
No dedicated test class for the `SimpleQueryUpdateEmitter`?

Reply:
It does not make sense to use `SimpleQueryUpdateEmitter` without `QueryBus`. That's why I combined those two in `SubscriptionQueryTest` to test them as a whole while using _subscriptionQuery_ feature.

Reply:
Not entirely convinced, but I do agree everything is tested through this method, so I am ok with it :+1: .
Thanks for adjusting the other comments!